packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js

```diff
@@ -264,4 +264,45 @@ MeetingInfoUtil.getRequestBody = (options) => {
   return body;
 };
 
+/**
+ * Helper function to parse the webex site/host from a URI string.
+ * @param {String} uri string (e.g. '10019857020@convergedats.webex.com')
+ * @returns {String} the site/host part of the URI string (e.g. 'convergedats.webex.com')
+ */
+MeetingInfoUtil.getWebexSite = (uri) => {
+  const exceptedDomains = ['meet.webex.com', 'meetup.webex.com', 'ciscospark.com'];
+  const site = uri?.match(/.+@(.+\..+\.*.)$/)?.[1];
+
+  return exceptedDomains.includes(site) ? null : site;
+};
+
+/**
+ * Helper function to return the direct URI for fetching meeting info (to avoid a redirect).
+ * @param {Object} options type and value to fetch meeting info
+ * @param {String} options.type One of [SIP_URI, PERSONAL_ROOM, MEETING_ID, CONVERSATION_URL, LOCUS_ID, MEETING_LINK]
+ * @param {Object} options.destination ?? value.value
+ * @returns {String} returns a URI string or null of there is no direct URI
+ */
+MeetingInfoUtil.getDirectMeetingInfoURI = (options) => {
+  const {
+    type, destination
+  } = options;
+
+  let preferredWebexSite = null;
+
+  switch (type) {
+    case _SIP_URI_:
+      preferredWebexSite = MeetingInfoUtil.getWebexSite(destination);
+      break;
+    case _LOCUS_ID_:
+      if (destination.info?.sipUri) {
+        preferredWebexSite = MeetingInfoUtil.getWebexSite(destination.info?.sipUri);
```

Comment:
info already has a webexSite info in it , We will not need the parsing logic here

Reply:
I don't understand.  distination.info.sipURL looks something like "10019857020@convergedats.webex.com" and I want it stripped down to "convergedats.webex.com".

Reply:
Oh, sorry, I see the webExSite property now.  Will change.